temperature/temperature.go

```diff
@@ -13,5 +13,8 @@ The temperature in Fahrenheit: 212.00
 */
 func main() {
 	fmt.Println("The program converts temperature from Celsius to Fahrenheit.")
-	// ++ Your code here! ++
+	fmt.Print("Enter the temperature in Celsius: ")
+	var cel float64
+	fmt.Scan(&cel)
+	fmt.Printf("The temperature in Fahrenheit: %.2f", (cel*9/5)+32)
```

Comment:
(same as in `speed.go`) It's better to assign the expression to the variable first, with the name reflecting the purpose of the expression.